app/(auth)/signup/future/page.tsx

```diff
@@ -1,13 +1,12 @@
 /* eslint-disable @typescript-eslint/no-misused-promises */
 import { ChevronRightSquareIcon } from "lucide-react";
-
 import Link from "next/link";
 
+import { signup } from "../../login/actions";
```

Comment:
@djsiddz Should you use absolute path here?

Reply:
Initially created the actions for login, but eventually added actions for signup and logout in this file. Moving this fine one folder up, to (auth) for now. Or I can split the actions and keep them closer to where they are used, so it will be a sibling import then. What do you think @sonalip9 ?